src/components/signIn/Form.tsx

```diff
@@ -1,23 +1,62 @@
 import 'components/signUp/Form.css'
 import 'components/signIn/Form.css'
+
 import smiliesIcon from 'assets/images/smilies.png'
-import { useState } from 'react';
+import axios from 'axios';
 import Paths from 'constants/paths';
+import Endpoints from 'constants/endpoints';
+import storeAPITokens from 'services/signIn/SignInHeaders';
+
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 
 
 const Form = () => {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [requestError, setrequestError] = useState(false);
```

Comment:
Following your states created above, this is how the name should be written:
```suggestion
  const [requestError, setRequestError] = useState(false);
```
WDYT?